### DISCUSSION TITLE
Officially port this theme on WebStorm

### DISCUSSION THREAD

**Discussion by calebeaires** [primer/github-vscode-theme#151] [Ideas]
I currently use both VSCode and WebStorm to develop different types of applications.

While at VSCode we have official support with official themes from Github, at Webstorm we can't find a theme with the same dedication, options, and quality.

I ask the GitHub team to consider porting this set of themes to the WebStorm.

Thank you very much.

**Comment by simurai** [comment]
In the past we created themes for [different editors](https://github.com/primer/github-syntax-theme-generator). But that project became a bit stale and isn't really maintained anymore. When launching [GitHub Codespaces](https://github.com/codespaces) that uses VS Code we created this theme so that the look and feel matches. But we don't really any plans to create more themes.

Have you looked into converting the existing VS Code theme to WebStorm? Maybe like [this](https://stackoverflow.com/questions/52238065/how-to-integrate-vs-code-theme-to-webstorm).

**Reply by simurai** [reply]
@calebeaires Oh.. just saw @jsynowiec's comment https://github.com/primer/github-vscode-theme/issues/26#issuecomment-660939835

> try those: https://gist.github.com/jsynowiec/f3570555174452bc8d73a1cda84db1e0

Which seems to be a port for different JetBrains IDEs.

**Reply by calebeaires** [reply]
Thanks, @simurai,

We have one or two themes there that fit only the editor part. Good ones! They are not immersive as those ones you've made to Code.

I'll try your link!

**Reply by simurai** [reply]
Ahh.. I see. Yeah, there are [a lot of colors](https://github.com/primer/github-vscode-theme/blob/e470bef97f7b976d20a480a55fdd34ef9ab379b6/src/theme.js#L23-L220) for the surrounding UI that might not be easy to map between editors since there isn't really a standard convention how to name these things.

**Reply by calebeaires** [reply]
I'm begging 😉
